profile/README.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to Open Horizon 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="https://github.com/Ayush7614/.github/blob/master/profile/open-horizon-color.png" width="55%" />
+</p>
+
+<p align="center">
+
+[![Slack](https://img.shields.io/badge/slack-@lf_edge--community-yellow.svg)](https://lfedge.slack.com/)
+[![Twitter](https://img.shields.io/badge/twitter-@lf_edge--twitter-9cf.svg)](https://twitter.com/lf_edge)
+[![YouTube](https://img.shields.io/badge/youtube-@lf_edge--youtube-red.svg)](https://www.youtube.com/channel/UCY7H1oSt8gvXNdXH9wrNq5Q)
+<a href="https://chat.lfx.linuxfoundation.org/#/welcome" alt="Element">
+<img src="https://img.shields.io/badge/Element-@element.svg?logo=element" /></a>
+</p>
+
+[Open Horizon](https://www.lfedge.org/projects/openhorizon/) is a platform for managing the service software lifecycle of containerized workloads and related machine learning assets. It enables autonomous management of applications deployed to distributed webscale fleets of edge computing nodes and devices without requiring on-premise administrators.
+
+### 🏂 Haven't tried Open Horizon yet ?
+Here are few things which we have developed in last couple of years.
+ - Try [anax](https://github.com/open-horizon/anax), Horizon agent control system .
+ - Checkout the [documentation website](https://github.com/open-horizon/open-horizon.github.io) to get started.
+ - Try [edge-sync-service](https://github.com/open-horizon/edge-sync-service), A subsystem for managing machine learning models at the edge.
+ - Try More Project on Github,[open-horizon](https://github.com/open-horizon).
+
+## Contact
+
+As a developer you can use Github issues and discussions on the respective projects to report bugs, request features, and discuss changes. We recommend using issues to connect with maintainers, contributors and other community members. As a user, you can use the [open horizon](https://www.lfedge.org/projects/openhorizon/) website to connect with us.
+
+You can reach out to us through the following channels as well:
+
+- [Mailing List](https://lists.lfedge.org/g/open-horizon) 
+
+- [Chat Server](https://chat.lfx.linuxfoundation.org/ )
+
+### Community driven development
+
+We love to build ❤ our product in open source.
+ - Please Join us such that we can make Open Horizon more robust and wider with your contribution.
```

Comment:
Let's add a statement at the bottom:

Open Horizon is a Stage Two: Growth project at [LF Edge](https://www.lfedge.org), an umbrella group within the Linux Foundation.